packages/react/src/components/badge/Badge.tsx

```diff
@@ -13,7 +13,7 @@ interface BadgeWithLabelProps extends BadgeBasicProps {
     label: string;
 }
 interface BadgeWithIconProps extends BadgeBasicProps {
-    icon: SvgName;
+    icon: Icon;

```

Comment:
Should we allow `React.ReactNode` instead?

I'm wondering if some people might want to use a custom icon in some cases

Reply:
Since the Badge component didn't allow `ReactNode` perviously, I doubt enforcing `Icon` here will be a problem. I honestly don't see a use case where someone would want a svg that is not in the iconography inside the badge, because it would be very small.